__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require("supertest");
+const app = require("../app");
+const seed = require("../db/seeds/seed");
+const data = require("../db/data/test-data");
+
+// beforeEach(() => {
+//   return seed(data);
+// });
```

Comment:
Why did you comment this out? Remember to close the DB connection after all as well